README.md

```diff
@@ -29,8 +29,10 @@ If you know a better way around this please make a [Issue ticket.](https://githu
 To be able to have your game working you need to make sure you have Steam running in the background. **SteamManager will print a Debug Message if it initializes correctly.**
 
 ## Client
+Before sending your game to your buddy make sure you have your **steamID64** ready. To find your **steamID64** the transport prints the hosts **steamID64** in the console when the server has started.
+
 1. Send the game to your buddy.
-2. Your buddy needs your **steamID64** to be able to connect. The transport shows your Steam User ID after you have started a server.

```

Comment:
> The transport shows your Steam User ID after you have started a server.

I felt this could be a bit confusing for someone who is new.

Reply:
Totally with you!